applications/asset_tracker_v2/src/events/debug_module_event.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _DEBUG_MODULE_EVENT_H_
+#define _DEBUG_MODULE_EVENT_H_
+
+/**
+ * @brief Debug module event
+ * @defgroup debug_module_event Debug module event
+ * @{
+ */
+
+#include "event_manager.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum debug_module_event_type {
+	/** Event carrying memfault data that should be forwarded via the configured cloud
+	 *  backend to Memfault cloud. Only sent if CONFIG_DEBUG_MODULE_USE_CUSTOM_TRANSPORT
+	 *  is enabled. Payload is of type @ref debug_module_memfault_data.
+	 */
+	DEBUG_EVT_MEMFAULT_DATA_READY,
+	DEBUG_EVT_ERROR
+};
+
+struct debug_module_memfault_data {
+	uint8_t buf[CONFIG_DEBUG_MODULE_MEMFAULT_CHUNK_SIZE_MAX];
+	size_t len;
+};
+
+/** @brief Debug event. */
```

Comment:
I dont think this short @brief is needed.

Reply:
Maybe not, I includeded it to align with the other module event docs in the application.